### PR TITLE
Reformatting Conversion, FeatureSelection and Image Analytics of Transform to Width 85

### PR DESCRIPTION
**Pull request by RadicalRayan**
Guidelines followed:
-85 characters per line
-Use 4 spaces for indentation
-Dot and open parentheses stay on same line as function
-If not a preexisting line under line that we break, add an extra line after it
-Don't indent comments
-Don't break a comment if it represents output
-Don't break links
-If applicable, break right before $
-Keep math op together

Fix for issue #3478

### CODE REVIEW

File: docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Conversion/ConvertType.cs [hunk 1]
```diff
@@ -20,17 +20,21 @@ public static void Example()
...
             var transformer = pipeline.Fit(data);
             var transformedData = transformer.Transform(data);
 
-            // Display original column 'Survived' (boolean) and converted column 'SurvivedInt32' (Int32)
-            var convertedData = mlContext.Data.CreateEnumerable<TransformedData>(transformedData, true);
+            // Display original column 'Survived' (boolean) and converted column 
+            // SurvivedInt32' (Int32)
+            var convertedData = mlContext.Data.CreateEnumerable<TransformedData>(
+                transformedData, true);
             foreach (var item in convertedData)
```
Comment (codemzs) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Conversion/ConvertType.cs]: new line between any two lines that were broken down into multiple lines but no new line between multiple breakdowns of a single line.

File: docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Conversion/Hash.cs [hunk 2]
```diff
@@ -24,30 +24,40 @@ public static void Example()
...
+            // the algorithms will mostly use the hashed values for further
+            // computations. The Hash method will preserve the mapping from the
+            // original values to the hashed values in the Annotations of the newly
+            // created column (column populated with the hashed values). 
+            //
+            // Setting the maximumNumberOfInverts parameters to -1 will preserve the
+            // full map. If that parameter is left to the default 0 value, the
+            // mapping is not preserved.
+            var pipeline = mlContext.Transforms.Conversion.Hash("CategoryHashed",
+                "Category", numberOfBits: 16, maximumNumberOfInverts: -1).Append(
```
Comment (codemzs) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Conversion/Hash.cs]: >.Append( [](start = 73, length = 8)

I would move .Append to a new line. to be clear we are adding a new transform.

File: docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Conversion/KeyToValueToKey.cs [hunk 3]
```diff
@@ -27,25 +27,40 @@ public static void Example()
 
             // A pipeline to convert the terms of the 'Review' column in 
             // making use of default settings.
-            var defaultPipeline = mlContext.Transforms.Text.TokenizeIntoWords("TokenizedText", nameof(DataPoint.Review))
-                .Append(mlContext.Transforms.Conversion.MapValueToKey(nameof(TransformedData.Keys), "TokenizedText"));
+            var defaultPipeline = mlContext.Transforms.Text.TokenizeIntoWords(
+                "TokenizedText", nameof(DataPoint.Review)).Append(mlContext
+                .Transforms.Conversion.MapValueToKey(nameof(TransformedData.Keys),
+                "TokenizedText"));
```
Comment (codemzs) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Conversion/KeyToValueToKey.cs]: please move .Append to a new line.

**Comment by RadicalRayan** [comment]
> 
> 
> Requesting changes. The readability of these reformat pull requests is _especially bad_. I left a bunch of notes in one of the earlier ones, but those notes do not cover all situations where this is worse than before.

Thank you for all of your comments and suggestions. We are going to review and check-in these PRs to address the issue of the 85 character limit. We will open a new issue with subsequent improvements to implement the style guides you had suggested.

File: docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Conversion/MapValueToKeyMultiColumn.cs [hunk 4]
```diff
@@ -30,19 +31,24 @@ public static void Example()
...
+
+            Console.WriteLine($" StudyTime   StudyTimeCategory   Course    " +
+                $"CourseCategory");
 
-            Console.WriteLine($" StudyTime   StudyTimeCategory   Course    CourseCategory");
             foreach (var featureRow in features)
-                Console.WriteLine($"{featureRow.StudyTime}\t\t{featureRow.StudyTimeCategory}\t\t\t{featureRow.Course}\t\t{featureRow.CourseCategory}");
+                Console.WriteLine($"{featureRow.StudyTime}\t\t" +
+                $"{featureRow.StudyTimeCategory}\t\t\t{featureRow.Course}\t\t" +
+                $"{featureRow.CourseCategory}");
```
Comment (sierralee51) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Conversion/MapValueToKeyMultiColumn.cs]: indent?

File: docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ImageAnalytics/ResizeImages.cs [hunk 5]
```diff
@@ -57,29 +62,41 @@ public static void Example()
 
         private static void PrintColumns(IDataView transformedData)
         {
-            Console.WriteLine("{0, -25} {1, -25} {2, -25} {3, -25}", "ImagePath", "Name", "ImageObject", "ImageObjectResized");
-            using (var cursor = transformedData.GetRowCursor(transformedData.Schema))
+            Console.WriteLine("{0, -25} {1, -25} {2, -25} {3, -25}", "ImagePath",
+                "Name", "ImageObject", "ImageObjectResized");
+            using (var cursor = transformedData.GetRowCursor(transformedData
```
Comment (sierralee51) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ImageAnalytics/ResizeImages.cs]: empty line after "Name" line?

**Pull request merged**